src/core.ts

```diff
@@ -24,7 +25,7 @@ export function getPageHtml(page: Page, selector = "body") {
     } else {
       // Handle as a CSS selector
       const el = document.querySelector(selector) as HTMLElement | null;
-      return el?.innerText || "";
+      return el?.innerText || el?.innerHTML || "";
```

Comment:
It occurs to me that this could introduce some undesired content to the crawler output since it would cover use cases like possibly grabbing scripts contents and white-spaces, for example:

```js
<div>
  <script>alert("Hello!");</script>
</div>
```

Would grab: `<script>alert("Hello!");</script>` as output (depending, of course, on the selectors config)

I'm likely not seeing the whole picture though, can you provide some practical examples you thought of covering @FTAndy ?

Reply:
@marcelovicentegc Yes, when I grab code block content from the tag  `textarea` in github project, I can't get the code content from the API `innerText` because it is a form controls element. It is a compatiable way to grab content in this case when the element is a form controls element.

I think the API `value` is more precise in this context because it will not transfrom symbol like `>` into `&gt`. What do you think?

Reply:
I do think the `value` API is a good idea, it is limited to input-type elements.